.github/workflows/ci.yml

```diff
@@ -12,24 +12,30 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: purescript-contrib/setup-purescript@main
+      - name: Set up a PureScript toolchain
+        uses: purescript-contrib/setup-purescript@main
         with:
           purescript: "unstable"
+          purs-tidy: "latest"
```

Comment:
Can you set this to a specific version? When we do breaking changes across core/contrib/node/web, it helps to keep these files consistent.

Reply:
Ok done.